backend/src/main/resources/db/migration/V145__Add_author_sort_name.sql

```
@@ -0,0 +1,5 @@
+ALTER TABLE author ADD COLUMN IF NOT EXISTS sort_name VARCHAR(255);
+ALTER TABLE author ADD COLUMN IF NOT EXISTS sort_name_locked BOOLEAN NOT NULL DEFAULT FALSE;
+
+UPDATE author SET sort_name = name WHERE sort_name IS NULL;
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)